sql_metadb/derived_tables/feesfines_accounts_actions.sql

```diff
@@ -1,44 +1,96 @@
     LEFT JOIN folio_users.groups__t AS ug ON uu.patron_group = ug.id
 ORDER BY fine_account_id, transaction_date;
 
+COMMENT ON COLUMN feesfines_accounts_actions.fine_account_id IS 'User fine/fee account id, UUID'
+
+COMMENT ON COLUMN feesfines_accounts_actions.fine_account_amount IS 'Amount of the fine/fee'
+
+COMMENT ON COLUMN feesfines_accounts_actions.fine_date IS 'Date and time the account of the fine/fee was created'
+
+COMMENT ON COLUMN feesfines_accounts_actions.fine_updated_date IS 'Date and time the account of the fine/fee was updated'
+
+COMMENT ON COLUMN feesfines_accounts_actions.fee_fine_id IS 'ID of the fee/fine'
+
+COMMENT ON COLUMN feesfines_accounts_actions.owner_id IS 'ID of the account owner'
+
+COMMENT ON COLUMN feesfines_accounts_actions.fee_fine_owner IS 'Owner of the account'
+
+COMMENT ON COLUMN feesfines_accounts_actions.fee_fine_type IS 'Fee/fine that is up to the desecration of the user'
+
+COMMENT ON COLUMN feesfines_accounts_actions.material_type_id IS 'ID of the material type of the item'
+
+COMMENT ON COLUMN feesfines_accounts_actions.material_type IS 'Material type of the item'
+
+COMMENT ON COLUMN feesfines_accounts_actions.payment_status IS 'Overall status of the payment/waive/transfer/refund/cancel'
+
+COMMENT ON COLUMN feesfines_accounts_actions.fine_status IS 'Overall status of the fee/fine'
+
+COMMENT ON COLUMN feesfines_accounts_actions.account_user_id IS 'ID of the user'
+
+COMMENT ON COLUMN feesfines_accounts_actions.transaction_id IS 'Fine/fee action id, UUID'
+
+COMMENT ON COLUMN feesfines_accounts_actions.transaction_amount IS 'Amount of activity'
+
+COMMENT ON COLUMN feesfines_accounts_actions.account_balance IS 'Calculated amount of remaining balance based on original fee/fine and what has been paid/waived/transferred/refunded'
+
+COMMENT ON COLUMN feesfines_accounts_actions.type_action IS 'Type of activity including the type of transaction'
+
+COMMENT ON COLUMN feesfines_accounts_actions.transaction_date IS 'Date and time the transaction of the fine/fee was created'
+
+COMMENT ON COLUMN feesfines_accounts_actions.transaction_location IS 'The service point where the action was created'
+
+COMMENT ON COLUMN feesfines_accounts_actions.transaction_information IS 'Number or other transaction id related to payment'
+
+COMMENT ON COLUMN feesfines_accounts_actions.operator_id IS 'Person who processed activity (from login information)'
+
+COMMENT ON COLUMN feesfines_accounts_actions.payment_method IS 'Overall status of the action-setting'
+
+COMMENT ON COLUMN feesfines_accounts_actions.user_id IS 'User UUID'
+
+COMMENT ON COLUMN feesfines_accounts_actions.user_patron_group_id IS 'UUID for user patron group'
+
+COMMENT ON COLUMN feesfines_accounts_actions.patron_group_name IS 'User patron group'
+
+COMMENT ON COLUMN feesfines_accounts_actions.signed_transaction_amount IS 'Identifies the type_action value that decreases the balance of the account and adds a - to those values'
```

Comment:
All comments are missing a ";" at the end

Reply:
Added ";" at the end of each comment.